src/Hedgehog/Gen.fs

```diff
@@ -551,3 +547,13 @@ module GenBuilder =
 module GenOperators =
     let (<!>) = Gen.map
     let (<*>) = Gen.apply
+
+open System
+
+/// In C# friendly manner, a generator for values and shrink trees of type 'a
+type Gen =
```

Comment:
Unsure if we really need a new `Gen` type since there's the existing [`GenLinqSupport`](https://github.com/hedgehogqa/fsharp-hedgehog/blob/ed75041fe59082153a0a659128c41cfabee2db10/src/Hedgehog/LinqSupport.fs) by @Porges.